kv/index_iter.go

```diff
@@ -237,6 +271,27 @@ func (c *kvIndex) Seek(txn Transaction, indexedValues []interface{}) (iter Index
 	return &indexIter{it: it, idx: c, prefix: c.prefix}, hit, nil
 }
 
+// CheckDuplicated check if the index is duplicated, only check the unique index
+func (c *kvIndex) CheckDuplicated(txn Transaction, indexedValues []interface{}, h int64) (duplicate bool, err error) {
```

Comment:
The implementation of this function is not the same as Exist, but I think supplying Exist may be better.